force-app/main/default/classes/ClassesObjectsMethods.cls

```diff
@@ -40,17 +40,29 @@ public with sharing class ClassesObjectsMethods {
      */
     public static List<String> practiceStringMethods() {
         // Initialize a String with 'Hello, Salesforce!'
-        
-        // Get the length of the String        
+        String greeting = 'Hello, Salesforce!';
+        // Get the length of the String
+        Integer stringLength = greeting.length();
+
         // Get a substring of the String of the first 5 characters
+        String stringSubstring = greeting.substring(0, 5);
+
         // Convert the String to lower case
+        String stringLowerCase = greeting.toLowerCase();
         
         // Return the list with the string data
+        List <String> result = new List<String> ();
+        
         // In the first position add the string length
+        result.add(string.valueOf(stringLength));
+
         // In the second position add the substring
+        result.add(stringSubstring);
+
         // In the third position add the string in lower case
+        result.add(stringLowerCase);
 
```

Comment:
Try to combine these lines using the addAll Method, create a new list and add the three strings in one line to the result List.